website/scripts/build-static-content.js

```diff
@@ -276,6 +276,33 @@ module.exports = {
 
               });//∞
 
+              // Modify images in the /articles folder
+              if (sectionRepoPath === 'articles/') {
+                // modifying relative links e.g. `../website/assets/images/articles/foo-200x300@2x.png`
+                htmlString = htmlString.replace(/((?<=(src))="(\.\/[^"]+|\.\.\/[^"]+)?")/g, (srcString)=>{
+                  let oldRelPath = srcString.match(/="(\.\/[^"]+|\.\.\/[^"]+)"/)[1];
+                  let referencedPageSourcePath = path.resolve(path.join(topLvlRepoPath, sectionRepoPath, pageRelSourcePath), '../', oldRelPath);
+                  // If the relative link goes to the image is in the website's assets folder (`website/assets/`) we'll modify the relative link
+                  // to work on fleetdm.com e.g. ('../website/assets/images/articles/foo-300x900@2x.png' -> '/images/articles/foo-200x300@2x.png')
+                  let isWebsiteAsset = referencedPageSourcePath.match(/(?<=\/website\/assets)(\/images\/(.+))/g)[0];
+                  if(!isWebsiteAsset) {
+                    // If the relative link doesn't go to the `website/assets/` folder, we'll throw an error.
+                    throw new Error(`Failed compiling markdown content: An article page has an invalid Image link ${srcString} at "${path.join(topLvlRepoPath, pageSourcePath)}".  To resolve, ensure this image has been added to 'website/assets/images/articles/' and link to it using a relative link e.g. '../website/assets/images/articles/foo-200x300@2x.png' OR a link to the image on fleetdm.com e.g. 'https://fleetdm.com/images/articles/foo-200x300@2x.png`);
+                  } else {
+                    return '="'+isWebsiteAsset+'"';
+                  }
```

Comment:
Suggestion: Inverse this if/then so that the heavy side is on the bottom.  (Makes it easier to read.)

i.e. it becomes
```
if (sectionRepoPath !== 'articles/') {
  //…
} else {
  //…
}